docs/access_rights/organization_account.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Organization account
+description: Create a specialized account for your team to encourage collaboration and manage permissions efficiently. Convert an existing account or create one from scratch.
+menuWeight: 8.1
+paths:
+    - access-rights/organization-account
+---
+
+# Organization account
+
+Apify's organization account allows you to keep your business account separate from your personal account. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
+
+At the same time, a separate organization account means teams no longer have to share the credentials of a single personal account. You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts seamlessly in just two clicks.
+
+You can start an organization account in two ways.
+* [Convert an existing account](#convert-an-existing-account) into an organization. If your actors and integrations are set up in a personal account, it is probably best to convert that account into an organization. This will preserve all your integrations but means you will need a new personal account.
+* [Create a brand-new account](#create-a-new-organization). If you don't have integrations set up yet, or if they are easy to change, you can create a new organization, preserving your personal account.
+
+> Any account can be converted into an organization. However, this option is only available on our **[[SPECIFY PLAN HERE]]** plan(s).
```

Comment:
This doesn't need to be here yet. But in the future, it will be possible for anybody to create an organization or convert an account into an organization BUT to add more users you will need a Startup plan (3 users) or business (5+ users, each extra seat paid)